draft-birkholz-scitt-software-use-cases.md

```diff
@@ -167,26 +167,34 @@ There is no standardized way to:
 * to check for the trustworthiness of a provider
 * to check the integrity of modifications or transformations done by a provider
 
-## Trustworthy Reference Values for After-The-Fact Analysis
+## Verifying Firmware Provenance for large and complex systems
+
+In contrast to operating systems or user space software components of a large and complex systems, firmware components are often already executed during boot-cycles before there is an opportunity to authenticate them.
+
+Authentication takes place, for example, by checking an accompanying signature or by creating a hash of measurements and comparing them to a reference value.
```

Comment:
@henkbirkholz I am not clear about the following line:

"and comparing them to a reference value."

You can create a Hash of running FW, but with whom you would compare it against ?

Reply:
This is where I believe a SCITT Trust Registry can provide value. This would give consumer the ability to query for trust declarations based on the SHA-256 hash value of a FW package, using somethings as simple as an http query that returns a trust score, i.e.: https://softwareassuranceguardian.com/SAGCTR_inquiry/getSAGScore?FileHash=94EDB27E1995370E9003EE0A0A12D0A7DE2E8DA5EB663C31A97CB4215AB171B6